homeassistant/components/syncthru/sensor.py

```diff
@@ -182,21 +168,23 @@ def entity_registry_enabled_default(self) -> bool:
 class SyncThruTonerSensor(SyncThruSensor):
     """Implementation of a Samsung Printer toner sensor platform."""
 
-    def __init__(self, syncthru, name, color):
+    def __init__(self, coordinator, name, color):
         """Initialize the sensor."""
-        super().__init__(syncthru, name)
+        super().__init__(coordinator, name)
         self._name = f"{name} Toner {color}"
         self._color = color
         self._unit_of_measurement = PERCENTAGE
         self._id_suffix = f"_toner_{color}"
 
-    def update(self):
-        """Get the latest data from SyncThru and update the state."""
-        # Data fetching is taken care of through the Main sensor
+    @property
+    def extra_state_attributes(self):
+        """Show all data returned for this toner."""
+        return self.syncthru.toner_status().get(self._color, {})
```

Comment:
It would be good to set the state attribute keys explicitly here so we csn check what they are and their formatting. Can be done in a follow up PR too.